### PR TITLE
Add Links for Dev, Operations and Additional Docs

### PR DESCRIPTION
**Pull request by afknapping**
all the links I collected so far

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,41 @@ A curated collection of projects and tools in the Urbit ecosystem.
...
+
+- [Beginning Hoon Introduction – hooniversity.org](https://hooniversity.org/beginning-hoon-introduction-2/)
+- [hoon-cheat-sheet(pdf)](https://storage.googleapis.com/media.urbit.org/docs/hoon-cheat-sheet-2020-07-24.pdf)
+- [timlucmiptev/gall-guide: Complete guide to Gall and Landscape apps on Urbit](https://github.com/timlucmiptev/gall-guide)
+- [Gall 101 – HTTP – hooniversity.org](https://hooniversity.org/gall-101-http/)
+- [CS 498MC • Martian Computing | martian-computing](https://davis68.github.io/martian-computing/)
+- [Nock for Everyday Coders, Part 1: Basic Functions and Opcodes](https://blog.timlucmiptev.space/part1.html)
+
+## Operations
+- [arthyn/taisho: A ship runner and manager for Urbit OS](https://github.com/arthyn/taisho)
```
Comment (jalehman) [inline README.md]:
```suggestion

- [arthyn/taisho: A ship runner and manager for Urbit OS](https://github.com/arthyn/taisho)
```

File: README.md [hunk 2]
```diff
@@ -3,4 +3,41 @@ A curated collection of projects and tools in the Urbit ecosystem.
 
 ## Games
 
-[Chess](https://sr.ht/~ray/urbit-chess) - play Chess over Urbit
+- [Chess](https://sr.ht/~ray/urbit-chess) - play Chess over Urbit
+
+## Apps
+
+- [rabsef-bicrym/tudumvc: todoMVC on Urbit](https://github.com/rabsef-bicrym/tudumvc/)
```
Comment (jalehman) [inline README.md]: This is actually learning material, not an app (learning material about building apps, with a todo list as the example project).

File: README.md [hunk 3]
```diff
@@ -3,4 +3,41 @@ A curated collection of projects and tools in the Urbit ecosystem.
...
+- [ryjm/srrs: a spaced repetition repetition system on urbit](https://github.com/ryjm/srrs)
```
Comment (jalehman) [inline README.md]:
```suggestion
- [srrs](https://github.com/ryjm/srrs) - a spaced repetition repetition system on urbit
```

Let's format things like this—by keeping the link text small and to the point, with the explanation following outside of the link. For example, see [here](https://github.com/avelino/awesome-go#authentication-and-oauth) or the format of the Chess link above.

**Review by jalehman — CHANGES_REQUESTED** [review]
Some formatting tweaks, otherwise looks good. Thank you for adding these!

**Comment by afknapping** [comment]
@jalehman PTAL, I think I adressed it all in a new commit

**Review by jalehman — APPROVED** [review]

**Pull request merged**